docs/concepts/models/sql_models.md

```diff
@@ -1,37 +1,40 @@
 # SQL models
 
-SQL models are the main types of models used by SQLMesh. SQL models consist of three sections:
+SQL models are the main type of models used by SQLMesh. These models can be defined using either SQL or Python.
```

Comment:
it should be types because of the are i think

Reply:
using either SQL or Python that generates SQL

Reply:
> it should be types because of the are i think

In my view, the previous form read like there are several _types_, where there's just one: that which represents these SQL models. I'm a bit rusty on my grammar though, so happy to revert if this is incorrect.

Reply:
> using either SQL or Python that generates SQL

Good point, I like this more

Reply:
> it should be types because of the are i think

Could also become «SQL models constitute the main type of models used by SQLMesh»

Reply:
type is probably better here - good catch